contrib/neon/Makefile

```diff
@@ -4,7 +4,14 @@
 MODULE_big = neon
 OBJS = \
 	$(WIN32RES) \
-	inmem_smgr.o libpagestore.o pagestore_smgr.o relsize_cache.o neon.o
+	inmem_smgr.o \
```

Comment:
leftover?

Reply:
Ah, never mind, sorry. I thought that `inmem_smgr.c` is deleted and `inmem_smgr.o` is leftover. But in fact `inmem_smgr.c` is still present and is used. Sorry for the noise.